filament/backend/src/vulkan/VulkanDriver.cpp

```diff
@@ -888,12 +895,52 @@ void VulkanDriver::beginRenderPass(Handle<HwRenderTarget> rth, const RenderPassP
         }
     }
 
-    const VkCommandBuffer cmdbuffer = mContext.commands->get().cmdbuffer;
     VulkanAttachment depth = rt->getSamples() == 1 ? rt->getDepth() : rt->getMsaaDepth();
+#ifdef VULKAN_DEBUG_REAL
```

Comment:
if `VULKAN_DEBUG_REAL` is one of ours, it should be `FILAMENT_VULKAN_DEBUG_REAL`

Also is it intended to be `READ` instead of `REAL`? (I don't understand what REAL means in this context)

Reply:
sorry leftover from my own debug.  Changed to `FILAMENT_VULKAN_VERBOSE`